flight_planning/v1/user_notification.yaml

```diff
@@ -0,0 +1,20 @@
+info:
+  title: Definitions used in flight planning interface for user notification.
+components:
+  schemas:
+    ConflictNotification:
+      type: object
+      description: Notification of conflict with new/modified operations.
+      required:
+        - notification_sent_at
+      properties:
+        notification_sent_at:
+          anyOf:
+            - $ref: './geotemporal.yaml#/components/schemas/Time'
+          description: Time at which the user or user's automated system was sent user notification about the conflict or intersection.
+        conflicts_count:
```

Comment:
By this same logic, couldn't the notification_sent_at also be "I don't know"? I'm nervous about overloading integers in this field or the time field too much.  Haven't thought this through completely, but seems better to just have these be optional fields? If the values are not known then they would just not be provided. This should cause the test to fail since the correct calculations would not be possible, but as you said maybe this is a valid response?

If both fields are optional, then what does that mean for this model?

Reply:
If this is the case, I'd still prefer something like an enum with strong definitions than these sentinel values.  If you needed another "sentinel value" in the future, you'd have to add a '-2' or something.

So a required `conflict_status` enum with initial values of `I_DONT_KNOW`, `NONE`, `SINGLE`, `MULTIPLE` and keep the conflict_count field as optional with a meaningful "count"... to me this may be better design if we have to map the testing capability to people in colorful shirts.

Reply:
I want to second the "I don't know" indication here.  To maximize compatibility, I think we should avoid attempting to differentiate between 0 and empty (this design practice is followed by all ASTM UTM standard APIs as well as existing InterUSS automated testing APIs).  But, I also think we need to be able to differentiate "The notification indicated 0 conflicts" from "The notification didn't mention anything about conflicts".  There are a few different ways to do this (e.g., add magic -1 value to indicate no conflicts, add another bool field named `indicates_conflicts`/`describes_conflicts`, change this field to be the enum Joey suggested, keep this field and add the new enum Joey suggested), but I don't think the current structure enables this differentiation.